src/bundling/InboxyStyler.js

```diff
@@ -66,29 +66,33 @@ class InboxyStyler {
     }
 
     /**
-     * For each bundle, disable bulk-archiving if any message outside of its bundle is selected.
+     * For each bundle, disable bulk-actions if any message outside of its bundle is selected.
      */
-    disableBulkArchiveIfNecessary() {
+    disableBulkActionsIfNecessary() {
         const selectedMessages = [].slice.call(
             document.querySelectorAll(Selectors.SELECTED));
         Object.values(this.bundledMail.getAllBundles()).forEach(bundle => 
-            this._updateBulkArchiveButton(bundle, selectedMessages));
+            this._updateBulkActionButtons(bundle, selectedMessages));
     }
 
     /**
-     * Enable/disable the bulk archive button for the given bundle.
+     * Enable/disable the bulk action buttons for the given bundle.
      */
-    _updateBulkArchiveButton(bundle, selectedMessages) {
+    _updateBulkActionButtons(bundle, selectedMessages) {
         const bundledMessageIds = new Set(bundle.getMessages().map(m => m.id));
         const allSelectedMessagesInBundle = !selectedMessages.some(
             m => !bundledMessageIds.has(m.id));
+        const bulkActionButtons = bundle.getBundleRow().querySelectorAll('span.action');
```

Comment:
Related to an earlier comment, `action` could be renamed to `bulk-action` (makes it easy to search for), and `span` is not needed here.